orangecontrib/prototypes/hierarchicalclusteringplus/distance/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .distance import (Distance, DistanceModel,
+                       Euclidean, Manhattan, Cosine, Jaccard,
+                       SpearmanR, SpearmanRAbsolute, PearsonR, PearsonRAbsolute,
+                       Mahalanobis, MahalanobisDistance, Hamming, Bhattacharyya)
```

Comment:
I think there's no need to have that separate from Orange. You can simply use:
`from Orange.distance import X, Y, Z`